test/parallel/test-cluster-disconnect.js

```diff
@@ -28,11 +28,11 @@ const net = require('net');
 if (cluster.isWorker) {
   net.createServer((socket) => {
     socket.end('echo');
-  }).listen(common.PORT, '127.0.0.1');
+  }).listen(0, '127.0.0.1');
 
-  net.createServer((socket) => {
+  const server  = net.createServer((socket) => {
     socket.end('echo');
-  }).listen(common.PORT + 1, '127.0.0.1');
+  }).listen(server.address().port + 1, '127.0.0.1');
```

Comment:
The reason we're trying to eliminate `common.PORT` from `parallel` is because it could have port collision with another test running at the same time using port `0`.

If you take some port and add a number to it, you still have this problem.

If possible, this should be `listen(0)` (which may require changes elsewhere in the test). If that's not possible or requires changes that make the test significantly more complicated, the other solution is to move the test as-is to `test/sequential` so that there won't be port collisions from other tests in `test/parallel`.